README.md

```diff
@@ -28,6 +28,8 @@
     <img src="https://img.shields.io/badge/X-000000.svg?style=for-the-badge&logo=X&logoColor=white" height=30px/></a>
   <a href="https://slack.flyte.org" alt="Twitter, formerly X logo label">
     <img src="https://img.shields.io/badge/Slack-Chat-pink?style=for-the-badge&logo=slack" alt="Flyte Slack label" /></a>
+  <a href="https://gurubase.io/g/flyte" alt="Flyte Guru on Gurubase.io">
```

Comment:
Thanks for considering and adding Flyte. I think training the LLM with public data is ok, but I don't think we want to redirect users from the readme to communication channels other to what the Flyte community maintains.